lib/mongoose_push/application.ex

```diff
@@ -159,6 +161,14 @@ defmodule MongoosePush.Application do
     end
   end
 
+  defp set_logformat(:fmt), do: set_loglevel(MongoosePush.Logger.Fmt)
+
+  defp set_logformat(:json), do: set_loglevel(MongoosePush.Logger.JSON)
```

Comment:
Are these calls correct? I'm wondering about the fact that `set_logformat` calls `set_loglevel` with a module, not sure if it's some Elixir magic, but it doesn't look intuitive to call later on `Logger.configure(level: MongoosePush.Logger.Fmt)` 🤔

Reply:
Thanks! Yeah, tests indeed would be nice, there weren't any for the Fmt, but I'll think of a way to test both formatters now.

Reply:
I would like to use `ExUnit.CaptureLog` to test the formats, but there was an issue with with that which caused the tests to fail. There were several fixes for the issues resulting in similar error, last one released only in `1.10.2`. I'm working on that, just messed something up on the way :D

Reply:
IMHO there is very little reason to use `ExUnit.CaptureLog`, especially that it's hard to use considering that it only works for "calling process", and this application has a lot of GenServes just running somewhere and doing stuff. Also, we don't need to test Logger's `console` backend - we can trust Elixir devs here :) If we write only formatter, we know what the contract between Logger's `console` backend and the formatter is, so unit tests for the formatter IMHO should be enough.

Reply:
I'll that, thanks:)